related?
```
======================================================================
ERROR: test suite for <module 'custom.icds_reports.tests' from '/mnt/commcare-hq-ro/custom/icds_reports/tests/__init__.pyc'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/vendor/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/__init__.py", line 87, in setUpModule
    table = metadata.tables[table_name]
KeyError: 'config_report_icds-cas_static-child_health_cases_a46c129f'
```

--- a/testsettings.py
+++ b/testsettings.py
@@ -102,3 +102,10 @@ LOGGING = {
     'version': 1,
     'loggers': {},
 }
+
+REPORTING_DATABASES = {
+    'default': 'default',
+    'ucr': 'default',
+    'icds-ucr': 'default',
+    'icds-test-ucr': 'default',
+}

